.dockerignore

```diff
@@ -0,0 +1,5 @@
+blog
```

Comment:
@tmigone can you please delete that? It was from my idea to add a blog section.

Reply:
Oh nice catch!

Reply:
@tmigone so you think we should keep the blog on the balena website=